test/gtest/expr_uses_test.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2017-present Barefoot Networks, Inc. 
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "gtest/gtest.h"
+#include "ir/ir.h"
+#include "ir/visitor.h"
+#include "lib/exceptions.h"
+#include "midend/expr_uses.h"
+
+TEST(expr_uses, expr_uses) {
```

Comment:
Thanks for adding a GTest! 👍 👍 👍

Reply:
My first gtest attempt -- I'm still feeling my way,